### DISCUSSION TITLE
osman+ 4.8.6 Android 14 no data import

### DISCUSSION THREAD

**Discussion by JoaSee** [osmandapp/OsmAnd#20717] [Q&A]
After I export to a file (5.6 GB) and reinstall OsmAnd+ 4.8.6 on a Samsung Galaxy S22 with Android 14, no data is shown; it is unavailable. The import starts, but there is no dialog window showing which data I want to import. After a few minutes, it finishes, and only the work address and home address are shown.
No maps, no tracks; everything was lost.
What did I do wrong?

Greetings,
Joachim

**Comment by albansuser** [comment]
I have found that only the export/import of Settings works. If one includes My Places or Resources the import doesn't work.

**Reply by JoaSee** [reply]
I tested it on another Samsung Galaxy S20 mobile with Android 13 and OsmAnd 4.8.5.
I can imported only tracks and settings, no resources.
Then I exported places and settings in two separate files.
On Samsung Galaxy S22 Android 14 with OsmAnd 4.8.6,
I can only import settings, but no tracks.

**Reply by mxg318** [reply]
I have the same issue, I cannot "restore from file" on a phone and a tablet that are both running Android 14. Works perfectly on older phones, I have tried 3 running Android 8 & 9. I contacted Osmand but they replied "unable to duplicate issue" then no further replies to my follow up emails. Seems this started with the app update earlier in August. Disappointing that they dont reply.
